test/ideal/sideal-test.jl

```diff
@@ -686,6 +686,11 @@ end
 
    @test ngens(std_hilbert(j, h, w, complete_reduction = true)) == 
          ngens(std(j, complete_reduction = true))
+   Qt,(t)= polynomial_ring(QQ, ["t"])
+   t=gen(Qt,1)
```

Comment:
Is the assignment `t = gen(Qt,1)` superfluous?  I think line 698 already assigns the correct value to `t`, right?